Sources/Logger/Loggers/FileLogger/FileLogger.swift

```diff
@@ -52,36 +59,56 @@ public class FileLogger: Logging {
     }
 
     var currentLogFileUrl: URL {
-        suiteName == nil ?
-            logDirURL.appendingPathComponent("\(currentLogFileNumber)").appendingPathExtension(logFilePathExtension) :
-            logDirURL.appendingPathComponent("extension-\(currentLogFileNumber)").appendingPathExtension(logFilePathExtension)
+        if let appName = appName {
+            return logDirURL
+                .appendingPathComponent("\(appName)-\(currentLogFileNumber)")
+                .appendingPathExtension(logFilePathExtension)
+        } else {
+            return logDirURL
+                .appendingPathComponent("\(currentLogFileNumber)")
+                .appendingPathExtension(logFilePathExtension)
+        }
     }
 
     public var levels: [Level] = [.info]
 
     // MARK: - Initializers
 
+    /// This is public constructor for FileLogger.
+    /// - Parameters:
+    ///   - appName: Name of the App used for appendingPathComponent of logDirURL.
```

Comment:
Z tohohle popisu fakt vubec nevim o co jde...
`String identifying an application instance (either the main application or one of its extensions). The identifier is used to separate logging files since each application instance has its dedicated log files.`

Reply:
Opraveno

Reply:
Opraveno.